in_toto/gpg/util.py

```diff
@@ -146,7 +146,7 @@ def compute_keyid(pubkey_packet_data):
   hasher.update(b'\x99')
   hasher.update(struct.pack(">H", len(pubkey_packet_data)))
   hasher.update(bytes(pubkey_packet_data))
-  return binascii.hexlify(hasher.finalize())
+  return binascii.hexlify(hasher.finalize()).decode("ascii")
```

Comment:
I wonder if for all of these instances we want an `errors='ignore'` or some sort of error handling...

Reply:
I think just blowing up is better. What we could do is to catch exceptions and re-raise with a redacted error message. Maybe worth a note in #126.